channels_redis/core.py

```diff
@@ -36,6 +53,7 @@ def _ensure_loop(self, loop):
             loop = asyncio.get_event_loop()
 
         if loop not in self.conn_map:
+            _wrap_close(loop, self)
```

Comment:
Let's highlight with a comment what's happening here

Reply:
Done. There's also an unrelated change  in `consistent_hash` to make the linter happy.